dallinger/heroku/Procfile

```diff
@@ -1,3 +1,4 @@
-web: python launch.py
-worker: python worker.py
-clock: python clock.py
+web: /home/cguardia/work/jazkarta/dallinger/bin/python launch.py
+worker: /home/cguardia/work/jazkarta/dallinger/bin/python worker.py
+clock: /home/cguardia/work/jazkarta/dallinger/bin/python clock.py
```

Comment:
Can you explain this part?

Reply:
Ugh, sorry for the noise. That got in there by mistake.